kolibri/core/assets/src/views/UserTypeDisplay.vue

```diff
@@ -26,7 +26,7 @@
     props: {
       userType: {
         type: String,
-        required: true,
+        required: false,
```

Comment:
if this is no longer required, should it have a default or explicit handling for `undefined`?